VultisigApp/VultisigApp/Services/UTXO/BlockchairService.swift

```diff
@@ -46,12 +46,14 @@ class BlockchairService {
         let data = try await Utils.asyncGetRequest(urlString: urlString, headers: [:])
         if let result = Utils.extractResultFromJson(fromData: data, path: "data.suggested_transaction_fee_per_byte_sat"),
            let resultNumber = result as? NSNumber {
-            let bigIntResult = BigInt(resultNumber.intValue)
+            // Apply the 1.5 factor to the fetched sats price
+            let bigIntResult = BigInt(Double(resultNumber.intValue) * 1.5)
```

Comment:
Don't change UTXO chain for now , as this is coming from blockchair , should be fine.  Also , usually UTXO chains take very long time to commit anyway